common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -405,7 +405,7 @@ protected void after() {
                     kubeClient().deleteContent(yaml);
                     // wait for ss to go
                     try {
-                        kubeClient().waitForResourceDeletion("statefulset", zkStatefulSetName);
+                        kubeClient().waitForResourceDeletion("statefulset", kafkaStatefulSetName);
```

Comment:
Why did you changed this? Isn't this intentionally waiting for deletion of Zookeeper because that is the last think being deleted when deleting cluster?

Reply:
That is strange. It looks like the ZK shutdown might be not happening. But I think this might easily break different tests because it means that this succeeds without properly deleting the cluster. I wonder if this starts breaking other tests instead.

Reply:
Yeah, the test was waiting for zk to shutdown but it never happened.
It passed all tests, so it probably does not break any other test.

Reply:
Well, if the ZK is never deleted, it means it is staying there and might cause problems in further tests.